### ISSUE TITLE
replace ParticleOutput by deal.II::Particle::DataOut

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#5063]
Is there anything missing in the deal.II version of this class? It was already available in  9.3...

https://github.com/geodynamics/aspect/blob/c9601fad8357f73b6c19bc396d6ef211a1585b03/include/aspect/postprocess/particles.h#L50

@gassmoeller ?

**Comment by gassmoeller** [comment]
Yes, the ASPECT version has the option to exclude certain particle properties from the output (by name). We have use cases with dozens of particle properties where we want to output only certain properties. I am not sure if that is something that would be useful for deal.II too. Maybe there are other (small) differences too. It would be useful to unify them though.

**Comment by bangerth** [comment]
@gassmoeller Want to keep this open?

Is outputting all particle properties an issue with disk space? Or why do you want to keep this option?

**Comment by gassmoeller** [comment]
Yes I think the disk space is an issue, especially for the models of @MFraters. Let's keep this open, I would still like to unify the ASPECT and deal.II classes.
